src/pages/workspaces/List.js

```diff
@@ -224,19 +224,19 @@ export const WorkspaceList = _.flow(
 )(class WorkspaceList extends Component {
   constructor(props) {
     super(props)
+    const { queryParams } = props
     this.state = {
-      filter: '',
+      filter: queryParams.filter || '',
       creatingNewWorkspace: false,
       cloningWorkspaceId: undefined,
       deletingWorkspaceId: undefined,
       sharingWorkspaceId: undefined,
       requestingAccessWorkspaceId: undefined,
-      accessLevelsFilter: [],
-      projectsFilter: [],
-      submissionsFilter: [],
-      includePublic: false,
-      tagsFilter: [],
-      ...StateHistory.get()
```

Comment:
Why not spread in `queryParams` here? Or if we're concerned about pollution, a `_.pick`ed version?